packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3078,6 +3082,11 @@ class _NotificationNode {
   }
 }
 
+bool _isProfileBuildsEnabledFor(Widget widget) {
+  return debugProfileBuildsEnabled || (debugProfileBuildsEnabledUserWidgets
+      && debugIsWidgetLocalCreation(widget));
+}
```

Comment:
the indentation doesn't match the scoping, this might be clearer as:
```dart
  return debugProfileBuildsEnabled
      || (debugProfileBuildsEnabledUserWidgets && debugIsWidgetLocalCreation(widget));
```

Reply:
done